src/segments/executiontime.go

```diff
@@ -223,3 +227,51 @@ func (t *Executiontime) formatDurationRound() string {
 	}
 	return fmt.Sprintf("%dms", t.Ms%second)
 }
+
+func (t *Executiontime) formatDurationLucky7() string {
+	// https://github.com/JanDeDobbeleer/oh-my-posh/issues/3970
+	// execution time will always be 7 characters long
+	// decimal point will be at the same location (3rd space or str[2])
+	// seconds and milliseconds will be aligned
+	// [m, s], [h, m], [d, h] will be aligned
+	if t.Ms < second {
+		//   999ms
+		// 1234567
+		return fmt.Sprintf("%5dms", t.Ms%second)
+	}
+
+	if t.Ms < minute {
+		// 12.34s
+		// 1234567
+
+		//  1.23s
+		// 1230 (= 1230ms)
+		// ^ use Sprintf pad left space
+		//  1230
+		// from here, just take 1, 23 of 230, and append s and ' '
+
+		result := fmt.Sprintf("%5d", t.Ms)
+
+		return result[:2] + "." + result[2:4] + "s "
+	} else if t.Ms < hour {
```

Comment:
the` else if` statements can be simple `if`'s as you return early.

Reply:
that is true

didn't notice it while refactoring it to early returns, thanks!